utils/kmstest.cpp

```diff
@@ -631,9 +693,19 @@ static void print_outputs(const vector<OutputInfo>& outputs)
 	for (unsigned i = 0; i < outputs.size(); ++i) {
 		const OutputInfo& o = outputs[i];
 
-		printf("Connector %u/@%u: %s\n", o.connector->idx(), o.connector->id(),
+		printf("Connector %u/@%u: %s", o.connector->idx(), o.connector->id(),
 		       o.connector->fullname().c_str());
-		printf("  Crtc %u/@%u", o.crtc->idx(), o.crtc->id());
+
+		for (const PropInfo &prop: o.conn_props)
+			printf(" %s=%" PRIu64 ",", prop.prop->name().c_str(),
+			       prop.val);
+
+		printf("\n  Crtc %u/@%u", o.crtc->idx(), o.crtc->id());
+
+		for (const PropInfo &prop: o.crtc_props)
+			printf(" %s=%" PRIu64 ",", prop.prop->name().c_str(),
+			       prop.val);
+
```

Comment:
For some reason this breaks the output:

```
# ./kmstest -c 1 -P DPMS=0 -r 640x480 -P zorder=2
Connector 1/@36: HDMI-A-1 DPMS=0,  Crtc 1/@40: 640x480 31.500 640/16/64/120 480/1/3/16 75 (75.00) 0xa 0x40
 zorder=2,  Plane 0/@31: 0,0-640x480
```

The trailing comma is also rather ugly there, and the format is different than with planes